pkg/git/git.go

```diff
@@ -44,6 +49,10 @@ func CloneRepo(ctx context.Context, url string, commit string, auth *Auth) error
 	if err != nil {
 		return err
 	}
+	_, err = shell.Execute(gitConfigCmd)
```

Comment:
It would be better to move this code `SetCredentialHelper()` func called from the main executor package after credential population, from a single responsibility point of view.